scripts/core/CONTRIB.rst

```diff
@@ -207,6 +207,16 @@ Experimental features *must* be defined in two new files, where
 *   ``scripts/core/exp-<feature>.yml`` defines the interface as an input to
     `Generating Source`_.
 
+To simplify this process please use the provided python script which will create
+these template files for you. You can then freely modify these files to 
+implement your experimental feature. 
+
+.. code-block:: console
+
+    $> cd scripts
+    $> python generate_exp.py <name-of-your-experimental-feature>
```

Comment:
could we not run this from the root of the repository instead?

also `$` instead of `$>` please.

Reply:
👍